final-project/README.md

```diff
@@ -5,6 +5,45 @@ This application allows users to sign up for an account, sign in, record and man
 
 The server and client are both written on Go, and run on Docker.
 
+## Project Directory
+```
+final-project/
+│   Taskfile.yml
```

Comment:
It's not because I've suggest and also using it, that it's a good fit for you.
Some people are using a Makefile, or some shell scripts, ...
If you want to use it, you will have to justify why it's a good choice for you :)